var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -53,9 +53,13 @@ class Zoltan(Package):
 
     variant('fortran', default=True, description='Enable Fortran support.')
     variant('mpi', default=True, description='Enable MPI support.')
+    variant('parmetis', default=False, description='Enable ParMETIS support.')
 
     depends_on('mpi', when='+mpi')
 
+    depends_on('parmetis@4:', when='+parmetis')
+    depends_on('mpi', when='^parmetis')
```

Comment:
parmetis will bring `mpi` as its dependency, so there is no need to have this line here.

Reply:
OK.  Thank you.

I have broken the long lines the Flake8 test found and will remove line 61.  I assume that I should force push these changes to my branch so they appear as one commit?

Reply:
>  I should force push these changes to my branch so they appear as one commit?

yes, if you squashed already the changes with the original commit, you need to push with `-f`.

Reply:
p.s. you probably should add `conflicts('+parmetis', when='~mpi')` as likely it does not make sense to build it with `~mpi+parmetis`.

Reply:
Thanks again. I just force pushed to my fork.  Locally executing `spack test` produces `918 passed, 75 skipped in 5830.47 seconds`.